app/controlplane/configs/config.devel.yaml

```diff
@@ -16,8 +16,8 @@ server:
     #   certificate: "../../devel/devkeys/selfsigned/controlplane.crt"
     #   private_key: "../../devel/devkeys/selfsigned/controlplane.key"
 
-nats_server:
-  uri: nats://0.0.0.0:4222
+# nats_server:
```

Comment:
Is this on purpose?

Reply:
Yes, a previous PR introduced this requirement in the dev environment while it's not a requirement to run the app.

I'll do it in another PR though